website/docs/d/project_api_key.html.markdown

```diff
@@ -44,5 +44,11 @@ In addition to all arguments above, the following attributes are exported:
 * `public_key` - Public key for this Organization API key.
 * `private_key` - Private key for this Organization API key.
 * `role_names` - Name of the role. This resource returns all the roles the user has in Atlas. The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#project-roles) describes the valid roles that can be assigned.
+### project_assignment
```

Comment:
This looks good, but do we need to make associated `project_assignment` doc edits to mongodbatlas_project_api_key**s** data source as well?

Reply:
yes, good point. Thanks. I will update it

Reply:
updated